lib/node_modules/@stdlib/_tools/modules/pkg-deps/bin/opts.json

```
@@ -1,20 +1,20 @@
 {
-	"string": [
-		"split",
-		"aliases"
-	],
-	"boolean": [
-		"help",
-		"version",
-		"no-builtins",
-		"no-walk"
-	],
-	"alias": {
-		"help": [
-			"h"
-		],
-		"version": [
-			"V"
-		]
-	}
+    "string": [
+        "split",
+        "aliases"
```

```suggestion
  "string": [
      "split",
      "aliases"
```

2 space indentation is expected in this here and elsewhere.